### ISSUE TITLE
Checkmark image on wallets page has incorrect alt text

### ISSUE THREAD

**Issue by JeremyRand** [bitcoin-dot-org/Bitcoin.org#3689]
On pages for individual wallets such as [this one](https://bitcoin.org/en/wallets/desktop/linux/electrum/), the [checkmark image](https://bitcoin.org/img/icons/checked.svg) in the Features list has the incorrect alt text `checked`.  This causes problems for users who use screen readers or who have SVG images disabled.  The alt text should be empty.
